url78eec0e0b6_Orikuro_vic_tgz-pJ8-net_nexon_vindictus_itemcomparer_inport_StaticImportJ8/src/net/nexon/vindictus/itemcomparer/inport/StaticImport.java

```diff
@@ -0,0 +1,2125 @@
+				15);
+
+		shoes.add(s);
+		gloves.add(g);
+		pants.add(p);
+		armors.add(a);
+		helms.add(h);
+	}
+
+	static void generateImperialSoft() {
+		String setname = "Imperial Guard Soft";
+		sets.add(setname);
+		List<ItemSetBonus> boni = new ArrayList<ItemSetBonus>();
+		boni.add(new ItemSetBonus(195, 0, 0));
+		boni.add(new ItemSetBonus(230, 0, 0));
+		boni.add(new ItemSetBonus(270, 25, 110));
+		boni.add(new ItemSetBonus(330, 45, 160));
+
+		ItemSet iset = new ItemSet(setname, boni);
+		iset.setUsefulChars(magic);
+
+		Shoes s = new Shoes(iset, "Shoes", 70, ItemTyp.LIGHT, 395, 0, 113, 22,
+				11);
+		Gloves g = new Gloves(iset, "Gloves", 70, ItemTyp.LIGHT, 372, 0, 106,
+				22, 9);
+		Pants p = new Pants(iset, "Pants", 70, ItemTyp.LIGHT, 470, 0, 132, 23,
+				12);
+		Helm h = new Helm(iset, "Helm", 70, ItemTyp.LIGHT, 442, 0, 106, 23, 9);
+		Armor a = new Armor(iset, "Armor", 70, ItemTyp.LIGHT, 532, 0, 140, 25,
+				13);
+
+		shoes.add(s);
+		gloves.add(g);
+		pants.add(p);
+		armors.add(a);
+		helms.add(h);
+	}
+
+	static void generateImperialHard() {
+		String setname = "Imperial Guard Hard";
+		sets.add(setname);
+		List<ItemSetBonus> boni = new ArrayList<ItemSetBonus>();
+		boni.add(new ItemSetBonus(202, 68, 0));
+		boni.add(new ItemSetBonus(222, 92, 0));
+		boni.add(new ItemSetBonus(297, 120, 0));
+		boni.add(new ItemSetBonus(375, 150, 0));
+
+		ItemSet iset = new ItemSet(setname, boni);
+		iset.setUsefulChars(fiona + " " + vella);
+
+		Shoes s = new Shoes(iset, "Shoes", 70, ItemTyp.HEAVY, 604, 93, 0, 21,
+				13);
+		Gloves g = new Gloves(iset, "Gloves", 70, ItemTyp.HEAVY, 600, 93, 0,
+				21, 11);
+		Pants p = new Pants(iset, "Pants", 70, ItemTyp.HEAVY, 749, 109, 0, 32,
+				14);
+		Helm h = new Helm(iset, "Helm", 70, ItemTyp.HEAVY, 700, 109, 0, 32, 11);
+		Armor a = new Armor(iset, "Armor", 70, ItemTyp.HEAVY, 850, 109, 0, 33,
+				15);
+
+		shoes.add(s);
+		gloves.add(g);
+		pants.add(p);
+		armors.add(a);
+		helms.add(h);
+	}
+
+	static void generateSkeletonWitch() {
+		String setname = "Skeleton Witch";
+		sets.add(setname);
+		List<ItemSetBonus> boni = new ArrayList<ItemSetBonus>();
+		boni.add(new ItemSetBonus(226, 0, 80));
+		boni.add(new ItemSetBonus(265, 0, 112));
+		boni.add(new ItemSetBonus(310, 0, 132));
+		boni.add(new ItemSetBonus(378, 0, 184));
+
+		ItemSet iset = new ItemSet(setname, boni);
+		iset.setUsefulChars(magic);
+
+		Gloves g = new Gloves(iset, "Gloves", 80, ItemTyp.PLATE, 759, 33, 127,
+				49, 13);
+		Shoes s = new Shoes(iset, "Shoes", 80, ItemTyp.PLATE, 773, 31, 127, 49,
+				13);
+		Pants p = new Pants(iset, "Pants", 80, ItemTyp.PLATE, 837, 38, 153, 53,
+				16);
+		Helm h = new Helm(iset, "Helm", 80, ItemTyp.PLATE, 810, 37, 135, 50, 15);
+		Armor a = new Armor(iset, "Armor", 80, ItemTyp.PLATE, 887, 39, 161, 55,
+				17);
+
+		g.setStamina(5);
+		s.setStamina(5);
+		p.setStamina(7);
+		h.setStamina(6);
+		a.setStamina(7);
+
+		shoes.add(s);
+		gloves.add(g);
+		pants.add(p);
+		armors.add(a);
+		helms.add(h);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 1130:1161, 1165:1196, 1200:1231, 1235:1266, 1634:1664.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.